code/_helpers/atoms.dm

```diff
@@ -18,3 +18,5 @@
 
 	return null//If we get here, the holder must be buried many layers deep in nested containers, or else is somehow contained in nullspace
 
+
```

Comment:
Was this intentional?

Reply:
I started writing a function to get pixel offsets, then ended up using forcemove instead and deleted the thing i wrote, guess i missed a space.